config.py

```diff
@@ -0,0 +1,8 @@
+import os 
+
+basedir = os.path.abspath(os.path.dirname(__file__))
+# path to the database
+SQLALCHEMY_DATABASE_URI = 'sqlite:///'+os.path.join(basedir,"app.db")
```

Comment:
use "app_db"or "app" instead of  "app.db" , it automatically creates the database and adds .db

Reply:
Thanks @matthewacha